packages/perseus/src/widgets/numeric-input/numeric-input.test.ts

```diff
@@ -47,6 +47,20 @@ describe("numeric-input widget", () => {
         expect(renderer).toHaveBeenAnsweredCorrectly();
     });
 
+    it("should reject an incorrect answer", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(question);
+
+        // Act
+        await userEvent.type(
+            screen.getByRole("textbox", {hidden: true}),
+            incorrect,
+        );
+
+        // Assert
+        expect(renderer).toHaveBeenAnsweredIncorrectly();
+    });
```

Comment:
This isn't new. I just re-organized the tests to group them better. If the first test was "accept correct answer", I would think the next test would be "reject an incorrect answer". This also enabled me to group the snapshot tests together.